### ISSUE TITLE
Inconsistency with Beam Emission 

### ISSUE THREAD

**Issue by Daniel-ler** [cherab/core#276]
Hi again, I have been having some inconsistent result in my CHERAB simulations, I pinpointed the issue to a beam emission not matching the expected value, however, that's where I am out of ideas. Below is a minimal working example that shows the issue.  

In detail; Beam is in single species plasma, I measure the beam emission perpendicular to the beam and though the beam axis. Now, I import the ADAS  coefficients using the CHERAB functionality, and it does not matter which way I put it there is ~%4 error:

1.  Either calculating the power which I am supposed to receive manually
2.  or calculation the beam density from the beam emission

I found that the error is independent of: 

- Beam energy/power/temperature
- plasma density/temperature
- tried different integration methods with no difference (although, this could be still an issue somehow) 
- Someone else reproduced the same error

I exhausted most of my ideas, there is one that I don't know how to check; to see if CHERAB is using the same BE coefficient when creating the spectrum as the one that I can import through    `adas.beam_emission_pec(...)`  

Is there anything that I am doing wrong or is there some deeper issue ?

```python
# External imports
import matplotlib.pyplot as plt
import numpy as np
from scipy.constants import electron_mass, atomic_mass
from raysect.core import Point3D, Vector3D
from raysect.optical import Ray,  World, Point3D, Vector3D
from raysect.primitive import Box

# Internal imports
from cherab.core.math import sample3d, Constant3D, ConstantVector3D
from cherab.core import Plasma, Maxwellian, Species, Beam
from cherab.core.atomic import elements
from cherab.core.atomic import Line, deuterium
from cherab.core.model import BeamEmissionLine
from cherab.core.model import SingleRayAttenuator
from cherab.core.model.beam.beam_emission import SIGMA_TO_PI, SIGMA1_TO_SIGMA0
from cherab.core.model.beam.beam_emission import PI2_TO_PI3, PI4_TO_PI3
from cherab.openadas import OpenADAS



# ########################### INIT CHERAB ################################### #

world = World()
adas = OpenADAS(permit_extrapolation=True)  # create atomic data source


# ########################### PLASMA SLAB ################################### #

# Parameters
slab_width = 2.0
slab_length = 10.0
d1_density = 4e19
d1_temperature = 1e3
d1_velocity = Vector3D(0,0,0)
e_density = d1_density
e_temperature = d1_temperature
e_velocity = d1_velocity

# Initialise plasma
plasma = Plasma(parent=world)
plasma.atomic_data = adas
plasma.geometry = Box(Point3D(0,-slab_width/2,-slab_width/2),
                      Point3D(slab_length, slab_width/2, slab_width/2))
e_distribution = Maxwellian(Constant3D(e_density),Constant3D(e_temperature),
                            ConstantVector3D(e_velocity), electron_mass)
plasma.electron_distribution = e_distribution
plasma.b_field = ConstantVector3D(Vector3D(0,0,0))

# Add deutrium
d1_distribution = Maxwellian(Constant3D(d1_density),Constant3D(d1_temperature),
                             ConstantVector3D(d1_velocity),
                             deuterium.atomic_weight * atomic_mass)
plasma.composition = [Species(deuterium,1,d1_distribution)]


# ########################### NEUTRAL BEAM ################################## #

# Parameters
beam_sigma = 0.025
beam_energy = 65000
beam_power = 3e6
beam_temperature = 20

# Inisialise beam
BES_model = BeamEmissionLine(Line(deuterium, 0, (3,2)),
                             sigma_to_pi=SIGMA_TO_PI,
                             sigma1_to_sigma0=SIGMA1_TO_SIGMA0,
                             pi2_to_pi3=PI2_TO_PI3, pi4_to_pi3=PI4_TO_PI3)

beam = Beam(parent=world)
beam.element = elements.deuterium
beam.plasma = plasma
beam.atomic_data = adas
beam.sigma = beam_sigma
beam.power = beam_power
beam.energy = beam_energy
beam.temperature = beam_temperature
beam.length = slab_length
beam.divergence_x = 0
beam.divergence_y = 0
beam.length = slab_length
beam.attenuator = SingleRayAttenuator(clamp_to_zero=True)
beam.models = [BES_model]
beam.integrator.step = 0.025
beam.integrator.min_samples = 10


# ########################### CHERAB EMISSION POWER ######################### #

# Paramameters
ray_z = 1            # How far along the beam are we looking?
ray_len = 2          # How far back from the beam is the observer
ray = Ray(origin=Point3D(ray_len/2,0,ray_z), bins=10000, min_wavelength=652,
          max_wavelength=660, direction=Vector3D(-ray_len/ray_len,0,0))
s = ray.trace(world)

dlambda = s.wavelengths[1] - s.wavelengths[0]
cherab_power = dlambda*np.sum(s.samples)


# ########################### MANUAL INTEGRATION ############################ #

# Get emission coefficient according to Cherab
bme_class = adas.beam_emission_pec(deuterium, deuterium, 1, (3,2))
bme = bme_class(beam_energy, e_density, e_temperature)

# Set up integration
integration_steps = 2000
emission = np.empty((integration_steps))
xpts = np.linspace(-ray_len,ray_len,integration_steps)
dx = xpts[1] - xpts[0]

# Step along sight-line
for i, xpt in enumerate(xpts):
    pt = Point3D(xpt,0,ray_z)
    beam_density = beam.density(pt.x, pt.y, pt.z)
    emission[i] = dx * bme * beam_density * e_density / (4*np.pi)

# Evaluate
manual_power = np.sum(emission)


# ########################### COMPARE ANSWERS ############################### #

percent_deviation = 100 * (cherab_power - manual_power) / manual_power

print("\nOUTPUT:")
print("Manual Integration \t: ",manual_power)
print("Cherab Integration \t: ",cherab_power)
print("Discrepancy (%) \t: ", percent_deviation)
```

The output: 
`OUTPUT:
Manual Integration 	:  0.37215472784373194
Cherab Integration 	:  0.3865335273805398
Discrepancy (%) 	:  3.8636616603311142
`

**Comment by vsnever** [comment]
Hi @Daniel-ler, thanks for reporting this. The bug is somewhere in the `BeamEmissionLine`. For some reason, the shape of this multiplet is not normalized to 1.

**Comment by vsnever** [comment]
It appears that the `intensity_s1` coefficient in `BeamEmissionMultiplet.add_line()` is assigned with wrong value. I'll make a PR with the fix.

**Comment by vsnever** [comment]
Fixed in #277.
